src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/CommentsIndentation/context/case3/Test.java

```diff
@@ -0,0 +1,22 @@
+package TreeWalker.CommentsIndentation;
+
+public class Test {
```

Comment:
Please add a comment above the class and add it to the diff:
```diff
+        // violation without a filter
public class Test {
```
This should cover CLASS_DEF scope.

Reply:
done